lark/parsers/earley.py

```diff
@@ -42,13 +44,21 @@ def __init__(self, parser_conf, term_matcher, resolve_ambiguity=True, debug=Fals
             if rule.origin not in self.predictions:
                 self.predictions[rule.origin] = [x.rule for x in analysis.expand_rule(rule.origin)]
 
-            ## Detect if any rules have priorities set. If the user specified priority = "none" then
+            ## Detect if any rules/terminals have priorities set. If the user specified priority = "none" then
             #  the priorities will be stripped from all rules before they reach us, allowing us to
```

Comment:
`stripped from all rules` and terminals?

Reply:
Also, it is the value `None`, not the value `"none"`.

Reply:
Right now the priority options are not applied to terminals. Is that something I should address in this PR or a separate one?

Reply:
Depends on how much additional work it is. If it is simple add it here, otherwise we can merge this and then come back.

Reply:
It should be pretty simple. I'll add it here.